app/experimenter/experiments/views.py

```diff
@@ -246,3 +246,7 @@ def view_fn(request, exception):
             return response
 
         return view_fn
+
+
+class ExperimentReportView(TemplateView):
+    template_name = "experiments/reporting.html"
```

Comment:
I think you should move this template into `app/experimenter/reporting/templates`?